### PR TITLE
Ignore Region logic

### PR DESCRIPTION
**Pull request by chinmay-browserstack**
Implementing Ignore Region for different types including ignore by id, xpath, selenium webelement and custom ignore region for POA.

### CODE REVIEW

File: percy/snapshot.py [hunk 1]
```diff
@@ -106,3 +111,10 @@ def percy_automate_screenshot(driver, name, **kwargs):
     except Exception as e:
         print(f'{LABEL} Could not take Screenshot "{name}"')
         print(f'{LABEL} {e}')
+
+def get_element_id_from_element(elements):
+    elements_array = []
+    for element in elements:
+        element_id = element.id
+        elements_array.append(element_id)
+    return elements_array
```
Comment (samarsault) [inline percy/snapshot.py:120]:
```python
def get_element_ids(elements):
    return [element.id for element in elements]
```

File: percy/snapshot.py [hunk 2]
```diff
@@ -87,16 +87,21 @@ def percy_automate_screenshot(driver, name, **kwargs):
     if not is_percy_enabled(): return
 
     try:
+        ignore_region_elements = \
+            get_element_id_from_element(kwargs.get("ignore_region_selenium_elements", []))
+        kwargs.pop("ignore_region_selenium_elements", None)
+
```
Comment (samarsault) [inline percy/snapshot.py:93]: Can we split this in multiple lines? Having `\` if not really necessary reads bad

File: percy/ignore_region.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# pylint: disable=R0903
```
Comment (chinmay-browserstack) [inline percy/ignore_region.py:1]: Remove this file

File: percy/snapshot.py [hunk 4]
```diff
@@ -87,16 +87,21 @@ def percy_automate_screenshot(driver, name, **kwargs):
     if not is_percy_enabled(): return
 
     try:
+        ignore_region_elements = kwargs.get("ignore_region_selenium_elements", [])
+        ignore_region_elements = get_element_ids(ignore_region_elements)
```
Comment (chinmay-browserstack) [inline percy/snapshot.py:91]: Refactor this

Comment (chinmay-browserstack) [inline percy/snapshot.py:91]:
```suggestion
        ignore_region_elements = get_element_ids(
            kwargs.get("ignore_region_selenium_elements", [])
        )
```

**Review by samarsault — APPROVED** [review]
LGTM

**Pull request merged**